wordpress-mobile/WordPress-iOS
# [Gutenberg] UBE - Implement custom JS code

gb-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2561
gutenberg PR: https://github.com/WordPress/gutenberg/pull/24658

This PR implements the changes from the previously mentioned PRs.

With these changes we are able to inject custom JS scripts to the Unsupported Block Editor. This is needed to do modifications related to WPCom, that are not allowed to do on gutenberg repo directly.

We inject `remove-nux.js` and `extra-localstorage-entries` already provided by `gutenberg-mobile`. This will remove the WPCom NUX screen when a new user opens gutenberg for the first time.

To test:

- Create a new wpcom account.
- On the newly created site, create a new gutenberg post.
- On HTML mode, paste this snippet (or any other unsupported block HTML code): ``
- Go back to visual mode.
- Open the unsupported block with the Unsupported Block Editor.
- Check that Gutenberg editor loads without the NUX UI covering it.


PR submission checklist:
